src/commcare_cloud/ansible/deploy_proxy.yml

```diff
@@ -106,6 +106,7 @@
         hosts_group: "{{ 'riakcs_new' if 'BLOB_DB_MIGRATING_FROM_S3_TO_S3' in localsettings else 'riakcs' }}"
         riakcs_proxy_port: "{{ temp_riakcs_proxy_port }}"
         site_config: {name: riakcs, vars_file: riakcs}
+      when: "'riakcs_new' in groups"
```

Comment:
It's not needed yet, but once this migration is complete (and there is no riak cluster) we'll want `when: "'riakcs' in groups"` above on the `riakcs site` task (line 91).

Reply:
Okay nice thinking. I'll add that now.

Reply:
Oh, you know what @millerdev? I think that this condition should be added to the `site_present` variable rather than a `when` clause: `set_site_present` both adds _and removes_ sites, based on the value of `site_present`. Using a `when` clause will at best _not add_ a site, but won't actively remove it when values change.

Along those lines, you'll see that "riakcs site" above already has `site_present: "{{ groups.get('riakcs') }}"`, so I think the issue behind your comment is already taken care of.

I'm going to go ahead and follow that pattern, adding `groups.get('riakcs_new')` to the `site_present` variable in this "riakcs new" task, and removing the `when`.